### PR TITLE
rename GetPrimarySkillsAndlocationsAndProjectNameAsync and add Projec…

### PR DESCRIPTION
**Pull request by alesyalis**
…tName

### CODE REVIEW

File: Recrutify/Recrutify.Host/Controllers/CandidatesController.cs [hunk 1]
```diff
@@ -116,10 +116,10 @@ public async Task<ActionResult> BulkSendEmailsWithTestAsync([FromBody] BulkSendE
         }
 
         [Authorize(Policy = Constants.Policies.AllAccessPolicy)]
-        [HttpGet("primary_skills_and_locations")]
-        public async Task<ActionResult<CandidateDTO>> GetPrimarySkillsAndlocationsAsync([FromQuery] Guid? projectId)
+        [HttpGet("primary_skills_and_locations_and_project_nane")]
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Host/Controllers/CandidatesController.cs:119]: na**m**e

Comment (elizavetapavliv) [reply Recrutify/Recrutify.Host/Controllers/CandidatesController.cs:119]: better to name - candidate_project_info

File: Recrutify/Recrutify.DataAccess/Models/CandidatesPrimarySkillsLocationAndProjectName.cs [hunk 2]
```diff
@@ -3,12 +3,14 @@
 
 namespace Recrutify.DataAccess.Models
 {
-    public class CandidatesPrimarySkillsAndLocation
+    public class CandidatesPrimarySkillsLocationAndProjectName
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.DataAccess/Models/CandidatesPrimarySkillsLocationAndProjectName.cs:6]: CandidatesProjectInfo, same for dto

Comment (alesyalis) [reply Recrutify/Recrutify.DataAccess/Models/CandidatesPrimarySkillsLocationAndProjectName.cs]: done

File: Recrutify/Recrutify.Services/Services/CandidateService.cs [hunk 3]
```diff
@@ -219,10 +219,13 @@ public async Task BulkSendEmailsWithTestAsync(BulkSendEmailWithTestDTO bulkSendE
             _sendQueueEmailService.SendEmailQueueForTest(candidates, project);
         }
 
-        public async Task<CandidatesPrimarySkillsAndLocationDTO> GetPrimarySkillsAndlocationsAsync(Guid? projectId)
+        public async Task<CandidatesProjectInfoDTO> GetPrimarySkillsAndlocationsAndProjectNameAsync(Guid? projectId)
         {
-           var primarySkillsAddLocations = await _candidateRepository.GetPrimarySkillAndLocationsAsync(projectId);
-           return _mapper.Map<CandidatesPrimarySkillsAndLocationDTO>(primarySkillsAddLocations);
+           var primarySkillsAddLocations = await _candidateRepository.GetPrimarySkillsAndlocationsAndProjectNameAsync(projectId);
+           var candidatesPrimarySkillsAndLocationAndProjectName = _mapper.Map<CandidatesProjectInfoDTO>(primarySkillsAddLocations);
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Services/Services/CandidateService.cs:225]: candidatesProjectInfo

**Review by elizavetapavliv — APPROVED** [review]

**Pull request merged**